students/hyeonwoo/users/models.py

```diff
@@ -0,0 +1,11 @@
+from decimal import MAX_EMAX
+from django.db import models
+
+class User(models.Model):
+    name               = models.CharField(max_length=45)
+    email              = models.CharField(max_length=45)
+    password           = models.CharField(max_length=45)
+    mobilephone_number = models.CharField(max_length=45)
```

Comment:
- email field에 unique 속성을 추가하셔서 유저가 중복 저장되는 문제가 발생하지 않도록 수정해주세요!(공식 문서에서 그 방법을 찾아보고 적용해주세요.)
- 이름과 관련된 필드는 아래를 참고해서 수정해 주세요.
    - id(계정) → `username`
    - name → `first_name`, `last_name`

```suggestion
    name               = models.CharField(max_length=45)
    email              = models.CharField(max_length=45)
    password           = models.CharField(max_length=45)
    mobile_number = models.CharField(max_length=45)
```

Reply:
네 알겠습니다! 수정 후 PR 하도록 하겠습니다.